tock/test_common.py

```diff
@@ -0,0 +1,78 @@
+import datetime
+import unittest
+
+from django.test import TestCase, override_settings
+from django.contrib.auth.models import User, Group
+
+from employees.models import UserData
+
+
+@override_settings(
+    PASSWORD_HASHERS=['django.contrib.auth.hashers.MD5PasswordHasher'],
```

Comment:
Out of curiosity, why MD5 and not SHA256 ([`django.contrib.auth.hashers.PBKDF2PasswordHasher`](https://docs.djangoproject.com/en/1.10/topics/auth/passwords/#included-hashers))?

Reply:
Oh, this is just for test cases, where we want the tests to run as quickly as possible (rather than be as secure as possible).  I believe PBKDF2 in particular is designed to be resistant to brute force attacks via computational cost, so it's great for production but actually makes tests run _really_ slowly.

I think that really new versions of Django might have introduced a better way to work around the slowness of PBKDF2 during testing, but if they did, I forget what it is...